### PR TITLE
Add a pip install melange build pipeline

### PR DESCRIPTION
**Pull request by dakaneye**
### Functional Changes

- [ ] This change can build all of Wolfi without errors (describe results in notes)

Notes:

### SCA Changes

- [ ] Examining several representative APKs show no regression / the desired effect (details in notes)

Notes:

### Linter

- [ ] The new check is clean across Wolfi
- [ ] The new check is opt-in or a warning

Notes:

### CODE REVIEW

File: pkg/build/pipelines/python/pipinstall.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Pip Install a Python package
+
+needs:
+  packages:
+    - busybox
```
Comment (vaikas) [inline pkg/build/pipelines/python/pipinstall.yaml:5]: Should this also require `pip` as a dependency? Like here:
https://github.com/chainguard-dev/melange/blob/main/pkg/build/pipelines/go/build.yaml#L5

Now, having said that, I wonder how this should be handled with the python versioning. I know @smoser has been working on that, so tagging him here.

Comment (dakaneye) [reply pkg/build/pipelines/python/pipinstall.yaml:5]: I thought about this, but there are 3 different pip packages, perhaps we add that as an input, defaulting to the most recent one?

**Comment by xnox** [comment]
@dakaneye we have this in-flight https://github.com/wolfi-dev/os/pull/20318 =)

File: pkg/build/pipelines/python/pipinstall.yaml (hunk 1 above)
Comment (smoser) [reply pkg/build/pipelines/python/pipinstall.yaml:5]: I'm guessing that @dakaneye found https://github.com/wolfi-dev/os/pull/20318/files after doing this.  (thanks for being thorough).

I think right now I'd not add this pipeline.  We may or may not end up adding one in the future, but our use of pip and python package installation is going through some changes.

**Review by smoser — CHANGES_REQUESTED** [review]

**Pull request closed without merging**